src/packet_analysis/protocol/gre/GRE.cc

```diff
@@ -46,12 +46,6 @@ bool GREAnalyzer::AnalyzePacket(size_t len, const uint8_t* data, Packet* packet)
 		return false;
 		}
 
-	if ( ! BifConst::Tunnel::enable_gre )
-		{
-		Weird("GRE_tunnel", packet);
```

Comment:
Was that a mechanism to notify you that "enable_gre" was disabled but there was GRE traffic. Seems fine to let go, but wonder what the motivation was here. If the analyzer is disabled, why would it create a weird? :-)

Reply:
I dunno, that code existed long before it got split out into the separate analyzer. This is from the release/3.2 branch in Sessions.cc:

```
	case IPPROTO_GRE:
		{
		if ( ! zeek::BifConst::Tunnel::enable_gre )
			{
			Weird("GRE_tunnel", ip_hdr, encapsulation);
			return;
			}
```

It was just leftover from when the code was moved, and probably should have been removed then.